src/pages/login/tests/login-page.spec.js

```diff
@@ -59,11 +63,33 @@ describe('Login page test', () => {
   });
 
   it('Should render Loader', () => {
-    storage = {
-      loginError: '',
-      userLoading: true
-    };
+    storage.userLoading = true;
     const component = shallow(<Login />);
     expect(component.exists(Loader)).toBe(true);
+    storage.userLoading = false;
+  });
+
+  it('should simulate change', async () => {
```

Comment:
I think title should be like this: "email should call useFormik method onChange". Same with others.